.github/actions/run-airbyte-ci/action.yml

```diff
@@ -115,7 +115,9 @@ runs:
         ls -la
         ls -la airbyte-python-cdk || echo "No airbyte-python-cdk directory"
         ls -laL ../airbyte-python-cdk || echo "No airbyte-python-cdk symlink"
-
+    - name: Install Java Environment
```

Comment:
clarification: does this run for all connector PR builds?

Reply:
Yes. But that could run conditionnally. I'm checking if its an easy fix ATM.
https://github.com/airbytehq/airbyte/pull/52033#discussion_r1924153466

Reply:
@davinchia I'm affraid there's not a straightforward quick way to conditionally run this.
We can't use rules changed files as nothing in file path determines that a connector is a java one.
We could run `airbyte-ci connectors --modified --language=java list` and check the output - but that could have its overhead too which might not be worth it given the speed of this step (~35sec).

Reply:
agreed with your conclusion, 35 secs seems fine.